.github/workflows/ci.yaml

```diff
@@ -56,19 +56,6 @@ jobs:
       - run: flutter pub get
       - run: flutter pub publish --dry-run
 
-  release:
-    if: github.event_name != 'pull_request'
-    runs-on: ubuntu-latest
-    permissions:
-      contents: write
-      pull-requests: write
-    steps:
-      - uses: google-github-actions/release-please-action@v3
-        with:
-          release-type: dart
-          changelog-notes-type: github
-          pull-request-title-pattern: 'chore: release v${version}'
-
```

Comment:
Is it normal that you removed this without replacing it? 🤔

Reply:
good catch, thanks! i copied the same release.yaml from the other repos but forgot to git add :facepalm: